examples/fabric/v4/cloudRouterConnectivity/cloudRouter2azure/two-connections/terraform.tfvars

```diff
@@ -0,0 +1,20 @@
+equinix_client_id = "Equinix_Client_Id"
```

Comment:
We talked about ignoring `.tfvars` files and (optionally) including a `.tfvars.example`.  Is that coming later?

Reply:
@ctreatma Yes, Tim has a PR ready for that and we plan to merge in next few days.